Formula/blast.rb

```diff
@@ -17,6 +17,16 @@ class Blast < Formula
   conflicts_with "proj", :because => "both install a `libproj.a` library"
 
   def install
+    # Fix shebang lines and JSON dependency
+    inreplace "c++/src/app/blast/update_blastdb.pl", "#!/usr/bin/perl", "#!/usr/bin/env perl"
```

Comment:
Add a comment as to why we use `JSON::PP` rather than `JSON`.

Reply:
Done.